enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -575,10 +581,22 @@ func NewPebbleCache(env environment.Env, opts *Options) (*PebbleCache, error) {
 		includeMetadataSize:         opts.IncludeMetadataSize,
 	}
 
-	versionMetadata, err := pc.databaseVersionMetadata()
+	versionMetadata, err := pc.DatabaseVersionMetadata()
 	if err != nil {
 		return nil, err
 	}
+	if newlyCreated && *opts.ActiveKeyVersion < 0 {
```

Comment:
I think opts.ActiveKeyVersion may be nil when initialized via the config?

Reply:
`SetOptionDefaults` (called on line 514) sets a default value for `opts.ActiveKeyVersion`.